framework/components/ecl-carousels/ecl-carousels.scss

```diff
@@ -0,0 +1,120 @@
+/**
+ * Carousel
+ * @define carousel ; weak
+ */
+
+.ecl-carousel {
+  align-items: stretch;
+  background-color: #000;
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+}
+
+@include ecl-media-breakpoint-up('xl') {
+  .ecl-carousel {
+    flex-direction: row;
+  }
+}
+
+.ecl-carousel__list-wrapper {
+  flex: 1;
+  overflow: hidden;
+  position: relative;
+}
+
+.ecl-carousel__controls {
+  margin: 0;
+}
+
+.ecl-carousel__list {
+  margin: 0;
+  white-space: nowrap;
+  width: 100%;
+}
+
+.ecl-carousel__item {
+  display: inline-block;
+  opacity: 0;
+  position: relative;
+  z-index: map-get($ecl-z-index, 'zero');
+}
+
+.ecl-carousel__item--showing {
+  opacity: 1;
+  z-index: map-get($ecl-z-index, 'highlight');
+}
+
+.ecl-carousel__btn {
+  background-color: #000;
+  border: 0;
+  color: #fff;
+  font-size: map-get($ecl-font-size, 'xxl');
+  position: absolute;
+  top: 50%;
+  z-index: map-get($ecl-z-index, 'navigation');
+
+  @include ecl-focus-outline-border();
+}
+
+.ecl-carousel__btn--previous {
+  left: 0;
+}
+
+.ecl-carousel__btn--next {
+  right: 0;
+}
+
+// JS will show only the necessary one by a data attribute.
+.ecl-carousel__image-information {
+  display: none;
+  text-align: left;
+}
+
+.ecl-carousel__live-region {
+  background-color: map-get($ecl-colors, 'grey-100');
+  color: #fff;
+  flex: 0;
+  min-width: 30%;
+  padding: map-get($ecl-spacing, 'l');
+}
+
+.ecl-carousel__meta {
+  margin-bottom: map-get($ecl-spacing, 'l');
+}
+
+// Every link has specific styling.
+.ecl-carousel__image-information a,
+.ecl-carousel__meta-link {
```

Comment:
Thanks - added, additional styles out of the icons removed. The states are not 1:1 match, but it's really ok for me.